src/style.css

```diff
@@ -1 +1,10 @@
 /* styles go here */
+
+.mainFieldset {
```

Comment:
`fieldset` will be enough
```suggestion
.fieldset {
```